### ISSUE TITLE
a little qusetion

### ISSUE THREAD

**Issue by arxhsyd123** [DamoSWL/Pairwise-Adversarial-Training#1]
Your code is quite neat and elegant. At least in my opinion, as a beginner, there are just some minor problems, which are harmless. However, if they are not corrected, your code will not work. For example, in my opinion, your code The imagelist class should inherit dataset instead of object, because the dataloader to be called next will report an error.
` images = ImageList(image_files, transform=transformer)
  images_loader = util_data.DataLoader(images, batch_size=batch_size, shuffle=False,
                                             num_workers=2)`
